recipes/default.rb

```diff
@@ -47,7 +45,21 @@
   cwd "#{node['timesync']['application_path']}/source"
 end
 
+execute 'create root user' do
+  command "npm run create-account -- -u root -p #{secret_key['root_pass']}"
+  environment environment
+  sensitive true
+  cwd "#{node['timesync']['application_path']}/source"
+end
```

Comment:
Does this need to be run only once or is it safe to run it on every run? Ideally it would be nice to only run this when it needs to so it speeds up normal chef runs. There are a variety of ways you could do it.